cdap-ui/app/cdap/components/Market/AllTab/index.js

```diff
@@ -79,7 +91,7 @@ export default class AllTabContents extends Component {
       </h4>
     );
     const empty = <h3>{T.translate('features.Market.tabs.emptyTab')}</h3>;
-    const entities = this.state.entities.map((e) => (
+    const entities = this.state.filterEntites.map((e) => (
       <MarketPlaceEntity key={e.id} entityId={e.id} entity={e} />
     ));
 
```

Comment:
We need to add another condition here. If `entities` is of length `0` we need to show the empty message.

Reply:
Done